java/src/main/java/ai/rapids/cudf/HostColumnVectorCore.java

```diff
@@ -115,29 +125,261 @@ Object getElement(int rowIndex) {
       int start = offHeap.offsets.getInt(rowIndex * DType.INT32.getSizeInBytes());
```

Comment:
Why is this not calling `getList` instead? I noticed that this does not deal with null handling either.

Reply:
It can and should. Updated.